rs/consensus/utils/src/lib.rs

```diff
@@ -283,11 +291,14 @@ pub fn find_lowest_ranked_proposals(pool: &PoolReader<'_>, h: Height) -> Vec<Blo
         .fold(
```

Comment:
nit: instead the `if` inside the `fold` closure, we could use `filter`, I think
```suggestion
        .filter(|proposal| !disqualified.contains(&proposal.signature.signer))
        .fold(
```

Reply:
done